changelog/v0.0.5/support-empty-schema.yaml

```diff
@@ -0,0 +1,5 @@
+changelog:
+  - type: FIX
```

Comment:
Is this a fix or a new feature? Should we increment the minor version?

Reply:
Great call, will update to a breaking change. The API is a collection of command line arguments, so it will compile, but it will fail with this change if a user tries to apply the option that was removed. Therefore, I think it should be considered a breaking change